src/js/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 3.18)
+
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 17)
+
+set(dist_folder "${CMAKE_SOURCE_DIR}/bin/")
+
+if(WIN32)
+    set(CMAKE_SHARED_LINKER_FLAGS /DELAYLOAD:NODE.EXE)
+    set(CMAKE_JS_LIB ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/node.lib)
+    set(CMAKE_JS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/win_delay_load_hook.cc)
+
+    set(CMAKE_JS_NODELIB_DEF ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/node-lib.def)
+    set(CMAKE_JS_NODELIB_TARGET ${CMAKE_JS_LIB})
+    set(DELAYIMP_LIB delayimp.lib)
+endif()
+
+project(genai_node_addon)
+
+# Add definitions
+add_definitions(-DNAPI_VERSION=8)
```

Comment:
> \# Add definitions

That's exactly what the function name says. You can describe in the comment what it is for instead.

Reply:
Fixed